### DISCUSSION TITLE
A central open-source repository for all kinds of plant models

### DISCUSSION THREAD

**Discussion by ilayn** [ilayn/harold#65] [Ideas]
This has come up in [reddit](https://www.reddit.com/r/ControlTheory/comments/jjtjg5/is_there_an_online_compendium_of_dynamic/) and I am curious how we can actually scan the literature as an open-source effort since academicians after 3 decades clearly not interested.

The obvious starting places are CAREX, DAREX etc. currently sitting in [SLICOT website](http://slicot.org/22-site/slicot-library/106-validation-of-control-software-and-benchmarking)

 Please chime in if you are willing to be a part of it and/or ideas about how we can comb the articles.

**Comment by skhatiwada013** [comment]
Hey, I am the OP of that post on reddit. This is a great idea. If we can have a community effort on this, I think it would be a great resource for all. Consider me in.

**Reply by ilayn** [reply]
Great. I'll start checking the licenses of those on SLICOT then we can create a repo somewhere
